File: app/src/main/java/com/example/whattowatch/ui/view/PersonView.java
package com.example.whattowatch.ui.view;

import com.arellomobile.mvp.MvpView;
import com.example.whattowatch.model.mymodel.MyPersonModel;

public interface PersonView extends MvpView {

    void showProgress(boolean isVisible);
    void showPersonInfo(MyPersonModel personModel);
    void showError();
}

File: app/src/main/java/com/example/whattowatch/ui/view/PersonView.kt
package com.example.whattowatch.ui.view

import com.arellomobile.mvp.MvpView
import com.example.whattowatch.model.mymodel.MyPersonModel

interface PersonView : MvpView {
    fun showProgress(isVisible: Boolean)
    fun showPersonInfo(personModel: MyPersonModel?)
    fun showError()
}